scripts/ci-entrypoint.sh

```diff
@@ -88,7 +88,11 @@ select_cluster_template() {
     fi
 
     if [[ "${EXP_MACHINE_POOL:-}" == "true" ]]; then
-        export CLUSTER_TEMPLATE="${CLUSTER_TEMPLATE/prow/prow-machine-pool}"
+        if [[ "${CLUSTER_TEMPLATE}" =~ "prow" ]]; then
+            export CLUSTER_TEMPLATE="${CLUSTER_TEMPLATE/prow/prow-machine-pool}"
+        elif [[ "${CLUSTER_TEMPLATE}" =~ "custom-builds" ]]; then
```

Comment:
we'll need to deal with CCM + machine pool too at some point

Reply:
Let's refactor the script in the near future. We should also address https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1372 at the same time.